src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/debugger/bsd/BsdDebuggerLocal.java

```diff
@@ -525,7 +525,7 @@ public long readJLong(long address)
     /** From the BsdDebugger interface */
     public long getAddressValue(Address addr) {
       if (addr == null) return 0;
-      return ((BsdAddress) addr).getValue();
+      return ((BsdAddress) addr).asLongValue();
```

Comment:
`asLongValue()` is declared in `Address`, so it should not be necessary to cast to a `BsdAddress`

Reply:
ok sure